doc/manual/plugins_global/saveimage.rst

```diff
@@ -3,12 +3,27 @@
 SaveImage (Save File)
 =====================
 
-This brings up :ref:`local-plugin-saveimage` local plugin from the global
-Plugins menu for convenience.
+.. image:: figures/saveimage_screenshot.png
```

Comment:
The screenshot needs to be moved to `manual/plugins_global/figures/` directory. Sphinx cannot find that image and it is causing Travis CI failure.